src/components/CMessageRow.vue

```diff
@@ -1,5 +1,8 @@
 <template>
-  <tr class="border-b border-gray-200 hover:bg-blue-100" @click="onToggle(messageId)">
+  <tr
+    class="border-b border-gray-200 hover:bg-blue-100 cursor-pointer "
```

Comment:
what i meant is more somerthing like this: https://www.ag-grid.com/javascript-grid-master-detail/ add the caret in the first column.